### PR TITLE
Ostap back connection

### PR DESCRIPTION
**Pull request by mariasykpotter**

### CODE REVIEW

File: client/src/components/AddPost/PostPost/AddPost.scss [hunk 1]
```diff
@@ -0,0 +1,79 @@
+@import '../../../styles/variables';
+
+
+.post-label {
+  font-size: 2em;
+  @media screen and (max-width:$break-small) {
```
Comment (anna-kuzii) [inline client/src/components/AddPost/PostPost/AddPost.scss]: pls add one empty row before media queries

File: client/src/components/AddPost/PostPost/AddPost.scss [hunk 2]
```diff
@@ -0,0 +1,79 @@
...
+    width: 20em;
+    height: 20em;
+  }
+  @media screen and (max-width:$break-small) {
+    width: 15em;
+    height: 15em;
+  }
+}
+
+.border-style {
```
Comment (anna-kuzii) [inline client/src/components/AddPost/PostPost/AddPost.scss]: look like it is a general class. Should we move it to the appropriate file? or it is class only for this component, maybe it is better than move it to its unique container

File: client/src/components/AddPost/PostPost/index.tsx [hunk 3]
```diff
@@ -1,15 +1,19 @@
...
         return (
-            // TODO Move style to scss
-            <div className='row d-flex pt-10 justify-content-lg-center
-                justify-content-sm-around justify-content-center'>
-                <div style={{height: '30em', width: '30em'}}>
-                    <img style={{height: '100%', width: '100%'}} src={this.props.croppedImage} alt=''/>
-                </div>
-            </div>
+        <Container>
+        <Row>
```
Comment (anna-kuzii) [inline client/src/components/AddPost/PostPost/index.tsx]: use tab pls. hard to read

File: client/src/styles/_variables.scss [hunk 4]
```diff
@@ -2,7 +2,13 @@ $color-red: #ff0000;
...
 $color-grey: #dcdcdc;
 $color-grey-dark: #aaaaaa;
+$empty-photo: #eeeeee;
 $color-snow: #f4f4f4;
 $color-black: #000000;
 $dark-red: #c40e0e;
+$color-button: #dc3545;
+$color1: #bdbdbd;
+$color2: #fafafa;
+$color3: #eeeeee;
```
Comment (anna-kuzii) [inline client/src/styles/_variables.scss]: u can take a name for color from here
http://chir.ag/projects/name-that-color/#001489

File: client/src/components/AddPost/PostPost/AddPost.scss [hunk 5]
```diff
@@ -0,0 +1,79 @@
+@import '../../../styles/variables';
+
+
```
[the remark this answers is not in the document]
Comment (mariasykpotter) [reply client/src/components/AddPost/PostPost/AddPost.scss]: What do you mean. We have variables.scss and style.scss separately.

File: client/src/components/AddPost/PostPost/AddPost.scss [hunk 6]
```diff
@@ -0,0 +1,79 @@
...
+  @media screen and (max-width:$break-small) {
+    width: 15em;
+    height: 15em;
+  }
+}
+
+.border-style {
+  color: $color1;
+  background-color: $color2;
+  border-color: $color3;
```
[the remark this answers is not in the document]
Comment (mariasykpotter) [reply client/src/components/AddPost/PostPost/AddPost.scss]: fixed

File: client/src/components/AddPost/PostPost/AddPost.scss (hunk 1 above)
Comment (mariasykpotter) [reply client/src/components/AddPost/PostPost/AddPost.scss]: fixed

File: client/src/components/AddPost/PostPost/AddPost.scss (hunk 2 above)
Comment (mariasykpotter) [reply client/src/components/AddPost/PostPost/AddPost.scss]: moved to a container  class drop-zone

File: client/src/components/AddPost/PostPost/index.tsx (hunk 3 above)
Comment (mariasykpotter) [reply client/src/components/AddPost/PostPost/index.tsx]: fixed

File: client/src/styles/_variables.scss (hunk 4 above)
Comment (mariasykpotter) [reply client/src/styles/_variables.scss]: Thanks.I used the exact names from this site.

File: client/src/components/AddPost/PostPost/AddPost.scss (hunk 5 above)
[the remark this answers is not in the document]
Comment (anna-kuzii) [reply client/src/components/AddPost/PostPost/AddPost.scss]: I mean about this file. Not AddPost.scss -- maybe styles.scss. Pls ask Volodia, how he named style files

**Pull request merged**